presto-atop/src/test/java/io/prestosql/plugin/atop/TestAtopSecurity.java

```diff
@@ -36,10 +36,10 @@
 
     @BeforeClass
     public void setUp()
-            throws IOException
+            throws Exception
     {
         Path atopExecutable = Files.createTempFile(null, null);
-        String path = Resources.getResource(getClass(), "security.json").getPath();
+        String path = new File(Resources.getResource(getClass(), "security.json").toURI()).getPath();
```

Comment:
shouldn't this be squashed with previous commit?

Reply:
Why would it be?

Reply:
What is the difference? What is returned by `Resources.getResource(getClass(), "security.json").getPath();`? Does it return `file://..`? And your change here is to cut it off? Am I right?

Reply:
`Resources.getResource`  / `getClass().getResouce` / `ClassLoader.getResouce` returns an `URL`.
`URL.getPath` returns ... well, path component of the URL, regardless whether it was `file:`, `jar:`, `http:` or `myurl:`. Also, `getPath` can be something slightly different then FS path, since (i think) it can contain URL escape sequences. Going thru `new File(URI)` addresses those issues.